Dockerfile

```diff
@@ -68,7 +70,12 @@ ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 # (e.g. in GitHub actions where $HOME points to /github/home).
 ENV XDG_CONFIG_HOME=/root/.config
 ENV XDG_CACHE_HOME=/root/.cache
-RUN curl -L https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 | bash && \
+RUN curl -fsSLo /tmp/helm.tgz https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz && \
+  echo "${HELM_SHA} /tmp/helm.tgz" | sha256sum -c - && \
+  tar -C /usr/local/helm -xzf /tmp/helm.tgz && \
+  rm /tmp/helm.tgz && \
+  export PATH="/usr/local/helm/linux-amd64:$PATH" && \
```

Comment:
```suggestion
  mkdir /usr/local/helm &&
  tar -C /usr/local/helm -xzf /tmp/helm.tgz --strip-components=1 linux-amd64/helm && \
  rm /tmp/helm.tgz && \
  export PATH="/usr/local/helm:$PATH" && \
```

Reply:
Manually committing as I included the `mkdir` in a recent forced push ✊

Reply:
Also should be `/usr/local/bin/helm` and hopefully drop the `export PATH`